http-clients/src/main/java/javax/ws/rs/PATCH.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package javax.ws.rs;
```

Comment:
Pretty sure Jersey blesses this approach -- the same approach is used in one of their own example repos for adding patch support (that example is a bit more expansive since it adds server-side support as well): https://github.com/jersey/jersey/tree/master/examples/http-patch.

This "transparent patch" support example was added in Jersey 2.6 (http://blog.dejavu.sk/2014/02/21/jersey-2-6-has-been-released-new-and-noteworthy/). Not sure why it's not part of the default implementation.